src/screens/Customers/AddCustomer.tsx

```diff
@@ -56,30 +58,37 @@ interface AddCustomerProps extends RouteComponentProps {
 }
 
 function AddCustomer(props: AddCustomerProps) {
+  const intl = useInternationalization(); 
   const { classes } = props;
   const history = useHistory();
   const allCustomerNumbers = useSelector(selectAllCustomersArray).map(customer => customer.customerNumber);
+  const meterTypeMap = new Map<string, string>(); // English keys
+  const revMeterTypeMap = new Map<string, string>(); // Native device language as keys
```

Comment:
Can you add a brief comment explaining why these are needed so it's clearer to new devs reading code?